### PR TITLE
s/craete/create/g

### PR DESCRIPTION
**Pull request by graingert**
### Description
it's a typo fix

### Checklist

This pull request is:

- [x] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: lib/sqlalchemy/dialects/postgresql/asyncpg.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 an ad-hoc event loop.  This mode of operation is of **limited use**
 and is for special testing scenarios only.  The mode can be enabled by
 adding the SQLAlchemy-specific flag ``async_fallback`` to the URL
-in conjunction with :func:`_sa.craete_engine`::
+in conjunction with :func:`_sa.create_engine`::
```
Comment (graingert) [inline lib/sqlalchemy/dialects/postgresql/asyncpg.py:31]: I'm somewhat surprised that sphinx didn't catch this as a no-member error as there's no documented `craete_engine` `:func:`

Comment (CaselIT) [reply lib/sqlalchemy/dialects/postgresql/asyncpg.py:31]: it probably logs is as warning, but I think there are too many to triage so they are mostly ignored :)

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision 3823b2552da2a7b7a345979ad6283d848c0ec7a5 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 3823b2552da2a7b7a345979ad6283d848c0ec7a5: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2351

**Comment by sqla-tester** [comment]
**Federico Caselli** (CaselIT) wrote:

Code-Review+1 Workflow+2

Everything looks ok

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2351

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Code-Review+2

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2351

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2351 has been **merged**. Congratulations! :)

**Pull request closed without merging**